_posts/en/newsletters/2020-07-29-newsletter.md

```diff
@@ -0,0 +1,153 @@
+  some RPC behavior changes resulting from those fixes, the planned
+  release provides compatibility with recent versions of [HWI][topic
+  HWI] and its support for hardware wallet firmware released for the
+  [fee overpayment attack][].
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19473][] net: Add -networkactive option
+  FIXME:bitschmidty
+
+- [Bitcoin Core #18044][] adds support for [BIP339][] witness txid
+  (wtxid) transaction announcements and requests using the `inv` and
+  `getdata` P2P protocol messages.  This eliminates a potential
+  inefficiency related to duplicate announcements of segwit-spending
+  transactions, making it cleaner to implement improvements such as
+  segwit v1 transactions ([taproot][topic taproot]) and [package
+  relay][topic package relay].  See our [extended summary][news104
+  bip339] of BIP339 for details.
+
+    After being merged into Bitcoin Core's master branch, this PR was
+    [discussed][meeting xscript] during the weekly Bitcoin Core
+    Development Meeting in relation to taproot activation.  It was noted
+    that the version of taproot merged into Bitcoin Core (see PR
+    [#17977][Bitcoin Core #17977]) will likely be based on wtxid relay.
+    This will make it more complicated to backport taproot to the
+    current 0.20.x branch of Bitcoin Core and so possibly ensure that
+    the earliest release of taproot activation logic will be in the
+    Bitcoin Core 0.21.x branch, whose major release is roughly
+    [expected][Bitcoin Core #18947] in December 2020.  A proposed
+    alternative for backporting taproot to the current 0.20.x branch,
+    either so miner signaling can start sooner or just so it's easier
+    for miners to upgrade, would be to only backport the logic for
+    verifying taproot transactions in blocks and not the wtxid
+    transaction relay logic.
+
+- [Eclair #1485][] adds support for [spontaneous payments][topic
+  spontaneous payments] using the same keysend protocol previously
+  implemented by LND (see [Newsletter #30][news30 spont]) and
+  C-Lightning (see Newsletters [#94][news94 keysend plugin] and
+  [#107][news107 keysend rpc]).  This merged PR supports both receiving
+  spontaneous payments (labeled as donations) and sending payments
+  using the new `sendWithPreimage` method.
+
+- [Eclair #1484][] adds low-level support for the changes to commitment
+  transactions for [anchor outputs][topic anchor outputs].  Not yet
+  added is higher-level support that will allow Eclair to negotiate
+  the use of anchor outputs with peers, but this early step does pass
+  all [proposed test vectors][BOLTs #688].
```

Comment:
Perhaps this should link directly to the comment: https://github.com/lightningnetwork/lightning-rfc/pull/688#issuecomment-656737250?